frontend/src/pages/notebookController/screens/server/NotebookServerRoutes.tsx

```diff
@@ -1,27 +1,17 @@
 import * as React from 'react';
-import { Switch, Route, useRouteMatch, Redirect } from 'react-router-dom';
+import { Routes, Route, Navigate } from 'react-router-dom';
 import NotebookServer from './NotebookServer';
 import NotebookControlPanelRedirect from './NotebookControlPanelRedirect';
 import SpawnerPage from './SpawnerPage';
 
 const NotebookServerRoutes: React.FC = () => {
-  const { path } = useRouteMatch();
-
   return (
-    <Switch>
-      <Route exact path={path}>
-        <NotebookServer />
-      </Route>
-      <Route exact path={`${path}/spawner`}>
-        <SpawnerPage />
-      </Route>
-      <Route exact path={`${path}/:username/(home|spawn)`}>
-        <NotebookControlPanelRedirect />
-      </Route>
-      <Route>
-        <Redirect to={path} />
-      </Route>
-    </Switch>
+    <Routes>
+      <Route path="/" element={<NotebookServer />} />
+      <Route path="/spawner" element={<SpawnerPage />} />
+      <Route path="/:username/:type" element={<NotebookControlPanelRedirect />} />
+      <Route path="*" element={<Navigate to="/notebookController" replace />} />
```

Comment:
We'll need to probably get constants involved here - FYI @dlabaj 

I didn't see an easy way to just reference "the path for this component"

Reply:
nvm, Juntao figured out a solution using `.`.